src/gui/advancedsettings.cpp

```diff
@@ -323,6 +326,14 @@ void AdvancedSettings::loadAdvancedSettings()
     spinBoxAsyncIOThreads.setValue(session->asyncIOThreads());
     addRow(ASYNC_IO_THREADS, tr("Asynchronous I/O threads"), &spinBoxAsyncIOThreads);
 #endif
+
+    // Checking Memory Usage
+    spinBoxCheckingMemUsage.setMinimum(1);
```

Comment:
You should add a special value to use libtorrent default and make it default in qBt. Probably 0 would be fine for that.

Reply:
@FranciscoPombal: 

[Quote from @zeule](https://github.com/qbittorrent/qBittorrent/pull/9373#discussion_r213650641): 
>You should add a special value to use libtorrent default and make it default in qBt. Probably 0 would be fine for that.

Reply:
Seems PR author has semi-abandon this, can we just accept it in current form and defer the request?
@zeule @glassez

Reply:
@Chocobo1 
Unfortunately it has been difficult for me to find the time to do anything meaningful with qBittorrent in general, including this PR. Perhaps merging this now is best so that someone else can then improve upon it and implement this enhancement. @zeule @glassez 

Sorry for the inactivity!

Reply:
>I think Yes, unless someone has strong reason to bury it finally.

Hmmm, even if we want to proceed, the blockage (change request) is still there. 
It seems the request can be dismissed in Github, however we don't have specific rules exercising it.